src/app/page.tsx

```diff
@@ -1,12 +1,28 @@
 "use client";
 
-import ExampleButton from "@components/ExampleButton";
+import StatsCard from "@components/StatsCard";
 
 export default function Home() {
   return (
     <div className="flex items-center justify-center h-screen">
-      {/* Example Button (ExampleButton.tsx in components folder) */}
-      <ExampleButton buttonText="Add User" />
+      <StatsCard
```

Comment:
Also I forgot but can you remove the components from this file before you merge?